include/z64animation.h

```diff
@@ -29,15 +44,15 @@ typedef struct {
     /* 0x06 */ u8 child;
     /* 0x07 */ u8 sibling;
     /* 0x08 */ Gfx* dLists[2]; // Near and far
-} LodLimb; // size = 0x10
+} LodLimb;                     // size = 0x10
```

Comment:
the formatter moved all of the sizes comments all the way to the side, can you put them back

Reply:
(this is one reason we only run the formatter on .c files, an unfortunate limitation)